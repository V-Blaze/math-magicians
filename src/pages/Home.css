.main-home {
  padding: 20% 15%;
  display: flex;
  justify-content: center;
  align-items: center;
}

.home-title {
  font-size: 1.8rem;
  font-style: italic;
}
