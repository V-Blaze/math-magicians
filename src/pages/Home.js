import React from 'react';
// stylesheet
import './Home.css';

const Home = () => (
  <div className="main-home">
    <h3 className="home-title">
      Math magicians is a website for all fans of mathematics.
      It is a Single Page App (SPA) that allows users to: Make simple calculations.
      Read a random math-related quote.
    </h3>
  </div>
);

export default Home;
