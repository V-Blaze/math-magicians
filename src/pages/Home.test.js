import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import React from 'react';
import Home from './Home';

it('snapshots', () => {
  const tree = renderer
    .create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
it('testing the heading inside home component', () => {
  render(<Home />);
  const heading = screen.getByText(/Math magicians is a website for all fans of mathematics./i);
  expect(heading).toBeInTheDocument();
});
