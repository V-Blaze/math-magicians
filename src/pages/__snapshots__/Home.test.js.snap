// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots 1`] = `
<div
  className="main-home"
>
  <h3
    className="home-title"
  >
    Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to: Make simple calculations. Read a random math-related quote.
  </h3>
</div>
`;
